netstandard/ref/System.cs

```diff
@@ -3550,9 +3670,14 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         public static System.String Concat(System.String str0, System.String str1, System.String str2, System.String str3) { throw null; }
         public static System.String Concat(params string[] values) { throw null; }
         public static System.String Concat<T>(System.Collections.Generic.IEnumerable<T> values) { throw null; }
+        public bool Contains(char value) { throw null; }
+        public bool Contains(char value, System.StringComparison comparisonType) { throw null; }
         public bool Contains(System.String value) { throw null; }
+        public bool Contains(System.String value, System.StringComparison comparisonType) { throw null; }
         public static System.String Copy(System.String str) { throw null; }
         public void CopyTo(int sourceIndex, char[] destination, int destinationIndex, int count) { }
+        public static System.String Create<TState>(int length, TState state, System.Buffers.SpanAction<char, TState> action) { throw null; }
```

Comment:
Where is System.Buffers.SpanAction introduced and why Action is not enough?

Reply:
@marek-safar 

> why Action is not enough?

`Span<T>` can't be used as type argument, to help ensuring they're always allocated on the stack. ([See this document for more details](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.2/span-safety.md#draft-language-specification).)

This means that `Action<Span<char>, TState>` would not be allowed.